Examples/Examples/ViewController.swift

```diff
@@ -26,7 +26,7 @@ class ViewController: MessageViewController, UITableViewDataSource, UITableViewD
         borderColor = .lightGray
 
         messageView.textViewInset = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 16)
-        messageView.font = UIFont.preferredFont(forTextStyle: .body)
```

Comment:
No big deal, but why did you change this?

Reply:
Realized this is already the default